src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence.mdx

```diff
@@ -12,36 +12,103 @@ redirects:
   - /docs/alerts-applied-intelligence/incident-intelligence/get-started-incident-intelligence
 ---
 
+import accountDiagram from 'images/incident-intelligence-account-diagram.png'
+
+import crossAccountDiagram from 'images/incident-intelligence-cross-account-diagram.png'
+
+import accountTeam from 'images/incident-intelligence-account-team.png'
+
+import environment from 'images/incident-intelligence-environment.png'
+
+import nerdgraph from 'images/incident-intelligence-nerdgraph.png'
+
 As part of [applied intelligence](/docs/introduction-new-relic-ai), incident intelligence helps you correlate your incidents and reduce noise in your environment. It gives you an overview of all your incidents, their sources, and related events.
 
 If you haven't already, [sign up for a New Relic account](https://newrelic.com/signup) to get started.
 
 <Callout variant="important">
   Pathways have been deprecated, and are replaced by [workflows](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/).
 
-  If you still have access to Pathways on New Relic One, reach out to your New Relic account representative and ask to be upgraded.
+  If you still have access to pathways, reach out to your New Relic account representative and ask to be upgraded.
 </Callout>
 
 ## Set up incident intelligence [#get-started]
 
-To enable incident intelligence, follow these steps. Afterwards, issues should start to appear in your issue feed.
+To enable incident intelligence and start seeing issues in  your issue feed, follow these steps.
 
 * [1. Configure your environment (one-time)](#1-configure-environment).
 * [2. Configure sources](#1-configure-sources).
 * [3. Configure destinations](#2-configure-destinations).
 * [4. Configure workflows](#3-configure-workflows).
 
-## 1. Configure your environment (one-time) [#1-configure-environment]
+## 1. Configure your environment [#1-configure-environment]
+
+An environment groups together a set of one or more accounts. This allows you to see cross-account actions such as correlation, configuration, and data in one place. Everyone on your team needs access to your environment account and be authorized to view the data in it.
+
+By default, when you create an account, it's added to its own environment. 
```

Comment:
i sent question about this, but basically: I think we need to define what we mean by 'account' here. Maybe something that explains "Your New Relic organization can have one or more accounts in it" and links to the explanation here of organizations and accounts? https://docs.newrelic.com/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/ 

When this says here "when you create an account, it's added to its own environment" it's not clear to me what the implication of that is. Is that just saying that each account starts out in its own environment, with each environment having a single account? And then you can adjust it from there and add other accounts to an account's environment? 

thinking about it more: does the 'environment' concept only pertain to applied intelligence? would it be helpful to explain explicitly what the environment is and if it might transcend applied intelligence or is firmly just an AI concept/use.

Reply:
Environment only pertains to applied intelligence, as far as I can tell.

Reply:
I'm trying to clarify this.